qa/integration-tests/src/test/java/io/camunda/it/client/QueryTest.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.it.client;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.client.api.command.CreateProcessInstanceCommandStep1;
+import io.camunda.zeebe.client.api.response.DeploymentEvent;
+import io.camunda.zeebe.client.api.response.ProcessInstanceEvent;
+import io.camunda.zeebe.client.api.search.response.SearchQueryResponse;
+import java.time.Duration;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Objects;
+import java.util.function.Function;
+import org.awaitility.Awaitility;
+
+abstract class QueryTest {
```

Comment:
🔧 We use this as a utility class (static methods only). Let's make it one then instead of it being an abstract class that should be implemented. Did you switch between making this a base class the other query ITs should extend and this being a utility class instead, maybe? 😅 
I would stick to a utility class like you already use 👍

Reply:
Thanks. Will use it as utility class.